src/styles/blocks/card.scss

```diff
@@ -0,0 +1,100 @@
+.card {
+  display: flex;
+  margin: 10px;
+  width: 200px;
+  flex-direction: column;
+  background-color: $card-color;
+  box-sizing: border-box;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 0;
+  }
+
+  &__title {
+    color: $main-text-color;
+    font-family: Roboto, sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 40px 16px 0;
+  }
+
+  &__code {
+    color: $secondary-text-color;
+    font-family: Roboto, sans-serif;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    margin-top: 4px;
+    margin-left: 16px;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 8px;
+  }
+
+  &__review_count {
+    margin: 0 16px 0 17px;
+    color: $main-text-color;
+    font-family: Roboto, sans-serif;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    text-align: right;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin: 22px 16px 0;
+  }
+
+  &__price_text {
+    margin: 0;
+    color: $secondary-text-color;
+    font-family: Roboto, sans-serif;
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+  }
+
+  &__price_count {
+    margin: 0;
+    font-family: Roboto, sans-serif;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    color: $main-text-color;
+  }
+
+  &__button {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 166px;
+    height: 42px;
+    background-color: $accent-color;
+    border: 1px solid $accent-color;
+    box-sizing: border-box;
+    border-radius: 5px;
+    font-family: Roboto, sans-serif;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    color: $card-color;
+    text-transform: uppercase;
+    text-decoration: none;
+    margin: 16px;
+  }
```

Comment:
The button styles are hardcoded. It would be better to make it more flexible, for example by using padding instead of width and height, and using a BEM modifier for the hover effect.